src/avfoundation.cs

```diff
@@ -796,31 +797,41 @@ interface AVAudioFormat : NSSecureCoding {
 
 		[Export ("isEqual:"), Internal]
 		bool IsEqual (NSObject obj);
+		
+		[iOS (10,0), TV (10,0), Watch (3,0), Mac (10,12)]
+		[NullAllowed, Export ("magicCookie", ArgumentSemantic.Retain)]
+		NSData MagicCookie { get; set; }
 	}
 
 	[Protocol, Model]
 	[BaseType (typeof (NSObject))]
 	interface AVAudio3DMixing {
+		[NoWatch]
```

Comment:
it seems everything is **not** available on watchOS so promote the attribute on the type itself. Otherwise we end up with a unusable (and confusing) stub on that unsupported platform.